targets/targets.json

```diff
@@ -2560,7 +2562,7 @@
         }
     },
     "DISCO_F413ZH": {
-        "components_add": ["QSPIF"],
+        "components_add": ["QSPIF", "SD", "FLASHIAP"],
```

Comment:
I don't think adding SD is correct
(on DISCO_F413ZH, SD connector is using SDIO not SPI)

Reply:
Hi @jeromecoutant,
So how can I approach this? should I delete it?